SEMINAR 8/controller.py

```diff
@@ -0,0 +1,20 @@
+import model
+import view
+import userinterface
+
+def button_click():
+    push_button = userinterface.menu()
+    if push_button == 1:
+        view.view_data(model.download_phonebook())
```

Comment:
У вас по-прежнему не очень привлекательный вид вывода меню
![image](https://user-images.githubusercontent.com/49741194/203688009-b44dfb71-e04c-41d4-a824-bf9920066d00.png)

Reply:
`model.download_phonebook()` ничего не возвращает, точнее аналога вращает только `None`, потому что в  ней не определен `return`. А функция `view.view_data` принимает один обязательный аргумент `self` Который там никак не используется. И по сему выходит, что функция `view.view_data` вообще не нужна. Все и так будет выводиться, потому что вывод у вас происходит в функции `view.view_data`. `import view` вообще можно убрать, так как функция `view_data` не приносит никакой пользы
```suggestion
        model.download_phonebook()
```